Model/Invoice.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2017 Classy Llama Studios, LLC
+ */
+
+namespace ClassyLlama\AvaTax\Model;
+
```

Comment:
Where magic getters and setters are used on your model you should include references to them in the DocBlock section of the class definition so that the IDE knows about them and can help catch any typo/validation issues. It also allows the IDE to provide auto complete support when using the class.

Example \ClassyLlama\AvaTax\Model\Queue:
```
/**
 * Queue
 *
 * @method string getCreatedAt() getCreatedAt()
 * @method string getUpdatedAt() getUpdatedAt()
 * @method int getStoreId() getStoreId()
```